pkgs/tools/misc/xsel/default.nix

```
@@ -1,15 +1,14 @@
-{stdenv, lib, fetchFromGitHub, libX11, autoreconfHook }:
+{ stdenv, lib, fetchFromGitHub, libX11, autoreconfHook }:
 
 stdenv.mkDerivation {
-  pname = "xsel-unstable";
-
-  version = "2019-08-21";
+  pname = "xsel";
+  version = "unstable-2020-05-27";
```

`unstable` should be part of `pname`. See: https://github.com/NixOS/nixpkgs/issues/68518
```suggestion
  pname = "xsel-unstable";
  version = "2020-05-27";
```

---

Oh, my bad. I thought it was prefered the other way around.